tensorflow/core/distributed_runtime/rpc/grpc_tensor_coding.cc

```diff
@@ -241,17 +241,13 @@ void EncodeTensorToByteBuffer(bool is_dead, const Tensor& val,
 
       // (E) Encode tensor data, but by sharing backing store
 
-      // TODO(jeff,sanjay): It'd be nice to avoid this TensorReference
-      // allocation, and instead get our hands on the underlying
-      // TensorBuffer object and just directly ref it here and unref
-      // it in unref_tensorreference.
-      TensorReference* ref = new TensorReference(val);
+      TensorBuffer* buf = DMAHelper::buffer(&val);
```

Comment:
Sorry there should be a `buf->Ref()`. It's fixed now.

`DMAHelper::buffer(&val)` does return a `const TensorBuffer*` so I will just make `buf` that type. It should be fine.